src/components/ItaliaTheme/View/ServizioView/ServizioCostiVincoli.jsx

```diff
@@ -21,19 +21,29 @@ const messages = defineMessages({
 
 const ServizioCostiVincoli = ({ content }) => {
   const intl = useIntl();
+  const sectionTtitleToRender = useMemo(() => {
```

Comment:
useMemo non lo userei qui... è un calcolo molto molto semplice da fare ad ogni render e di solito in questi casi è sconsigliato introdurre l'overhead di useMemo

Reply:
C'è anche un typo nel nome della variabile, la doppia t.. quasi quasi intanto che ci si ripassa vale la pena sistemarlo